torchmultimodal/models/omnivore.py

```diff
@@ -8,20 +8,53 @@
 from typing import Callable, List, Optional
 
 import torch
+import torchmultimodal.utils.common as common_utils
 from torch import nn
-from torchmultimodal.architectures.omnivore import OmnivoreArchitecture
 from torchmultimodal.modules.encoders.swin_transformer_3d_encoder import (
     PatchEmbed3d,
     SwinTransformer3d,
 )
 
 _OMNIVORE_PRETRAINED_URLS = {
-    "swin_t": "https://download.pytorch.org/models/omnivore_swin_t-5b532aca.pth",
-    "swin_s": "https://download.pytorch.org/models/omnivore_swin_s-b64cc260.pth",
-    "swin_b": "https://download.pytorch.org/models/omnivore_swin_b-c2a4d126.pth",
+    "swin_t_encoder": "https://download.pytorch.org/models/omnivore_swin_t_encoder-b7e39400.pth",
+    "swin_s_encoder": "https://download.pytorch.org/models/omnivore_swin_s_encoder-40b05ba1.pth",
+    "swin_b_encoder": "https://download.pytorch.org/models/omnivore_swin_b_encoder-a9134768.pth",
+    "swin_t_heads": "https://download.pytorch.org/models/omnivore_swin_t_heads-c8bfb7fd.pth",
+    "swin_s_heads": "https://download.pytorch.org/models/omnivore_swin_s_heads-c5e77246.pth",
+    "swin_b_heads": "https://download.pytorch.org/models/omnivore_swin_b_heads-3c38b3ed.pth",
 }
 
 
+class OmnivoreArchitecture(nn.Module):
+    """Omnivore is a model that accept multiple vision modality.
+
+    Omnivore (https://arxiv.org/abs/2201.08377) is a single model that able to do classification
+    on images, videos, and single-view 3D data using the same shared parameters of the encoder.
+
+    Args:   encoder (nn.Module): Instantiated encoder.
```

Comment:
start a new line for the list of args and inputs?

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
Thanks for noticing!